lib/models/messages/twitch/eventsub_configuration.dart

```diff
@@ -0,0 +1,111 @@
+import 'package:flutter/material.dart';
+
+class EventSubConfigurationModel extends ChangeNotifier {
+  bool showFollowEvent = true;
```

Comment:
I think it would be better if the default config only enabled the less "spammy" events.